plugins/generator-1.20.1/forge-1.20.1/procedures/strike_lightning.java.ftl

```diff
@@ -2,6 +2,6 @@
 if (world instanceof ServerLevel _level) {
 	LightningBolt entityToSpawn = EntityType.LIGHTNING_BOLT.create(_level);
 	entityToSpawn.moveTo(Vec3.atBottomCenterOf(${toBlockPos(input$x,input$y,input$z)}));
-	<#if (field$effectOnly!false)?lower_case == "true">entityToSpawn.setVisualOnly(true)</#if>;
+	<#if (field$effectOnly!false) == "TRUE">entityToSpawn.setVisualOnly(true)</#if>;

```

Comment:
Here it would be good to also specify correct false value "FALSE" for fallback. Check if this applies to any other parts